metricbeat/module/ceph/_meta/output-examples/status.json

```diff
@@ -0,0 +1,101 @@
+{
+   "health":{
+      "health":{
+         "health_services":[
+            {
+               "mons":[
+                  {
+                     "name":"prcdsrvv1619",
+                     "kb_total":6281216,

```

Comment:
Please check our naming conventions: https://www.elastic.co/guide/en/beats/libbeat/5.1/event-conventions.html

Whenever possible we should no use nested arrays. I think we need to have a closer look at the data set and potentially resturcture it to fit into metricbeat.